manifests/rhoai/params.env

```diff
@@ -1 +1,2 @@
 odh-kubeflow-trainer-controller-image=quay.io/opendatahub/trainer:v2.1.0
+odh-kubeflow-trainer-universal-workbench-image=quay.io/opendatahub/odh-training-th03-cuda128-torch28-py312-rhel9:latest
```

Comment:
@sutaakar just to clarify - will this value be essentially coming from related images at some point?